gestalt/vault.py

```diff
@@ -31,8 +31,8 @@ def __init__(self,
                 with role and jwt string from kubernetes
         """
         self._scheme: str = scheme
-        self.dynamic_token_queue: asyncio.Queue[Any] = asyncio.Queue(maxsize=0)
-        self.kubes_token_queue: asyncio.Queue[Any] = asyncio.Queue(maxsize=0)
+        self.dynamic_token_queue: Queue = Queue()  # type: ignore
```

Comment:
Any reason we have type ignores here?

Reply:
it was for the type of messages in the queue, i.e. should be `Queue[Tuple[str, str, str]]`. This passed `mypy` but broke a python version syntax so I just removed the mypy check and removed the type. 

Interestingly, adding `Queue[Tuple[str, str, str]]` passes both in tests as well as mypy. However, line 156 https://github.com/clear-street/gestalt/blob/awheelock/sc-61051/gestalt-resolve-warning-coroutine-is-not/gestalt/vault.py#L156 it will still fail mypy if I do not provide a type, but conflicts with python syntax (i.e. failing tests) if i add the type.